core/webviz/_webviz.py

```diff
@@ -28,11 +28,12 @@ def escape_all(html):
 
 class Page(object):
     """
-    A Page in the webviz. Contains :class:`PageElement` s. In order to be
-    rendered the :class:`Page` should be added to a :class:`Webviz`.
+    Container for :py:class:`PageElement` instances. In order to be
+    rendered the :py:class:`Page` should be added to a
+    :py:class:`Webviz` instance.
 
     :param title: String. A title for the page.
-    :param icon: `Optional Parameter` String.
+    :param icon: `Optional parameter`. String.
```

Comment:
This parameter is underspecified, should read something like: "The name of a icon provided by the theme".

Reply:
Similarly SubMenu needs a better description of icon.